ffc-netcore-parent/runtime/Dockerfile

```diff
@@ -1,2 +1,7 @@
-ARG NETCORE_VERSION
-FROM mcr.microsoft.com/dotnet/core/aspnet:${NETCORE_VERSION}-alpine
```

Comment:
We should also include the .Net environment, ie `ENV ASPNETCORE_ENVIRONMENT=production`  the development Dockerfiles should have `ENV ASPNETCORE_ENVIRONMENT=development`